m/configLocalAccess.m

```diff
@@ -0,0 +1,51 @@
+function db_access = configLocalAccess()
+%CONFIGDBACCESS  Configure data base access parameters for glider processing.
+%
+%  Syntax:
+%    DB_ACCESS = CONFIGDBACCESS()
+%
+%  Description:
+%    DB_ACCESS = CONFIGDBACCESS() should return a struct with the access
+%    parameters for the data base to be used during the glider processing
+%    as needed by the function GETDEPLOYMENTINFODB.
+%
+%  Notes:
+%    Edit this file filling in the parameters of your data base.
+%    Once configured, this file will contain private data, so you should not
+%    make it public.
+%
+%  Examples:
+%    db_access = configDBAccess()
+%
+%  See also:
+%    GETDEPLOYMENTINFODB
+%
+%  Authors:
+%    Joan Pau Beltran  <joanpau.beltran@socib.cat>
+
+%  Copyright (C) 2013-2015
+%  ICTS SOCIB - Servei d'observacio i prediccio costaner de les Illes Balears
+%  <http://www.socib.es>
+%
+%  This program is free software: you can redistribute it and/or modify
+%  it under the terms of the GNU General Public License as published by
+%  the Free Software Foundation, either version 3 of the License, or
+%  (at your option) any later version.
+%
+%  This program is distributed in the hope that it will be useful,
+%  but WITHOUT ANY WARRANTY; without even the implied warranty of
+%  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%  GNU General Public License for more details.
+%
+%  You should have received a copy of the GNU General Public License
+%  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+  error(nargchk(0, 0, nargin, 'struct'));
+
+% $$$   db_access.url  = 'jdbc:postgresql://mydb.myportal.mydomain:5433/my_data_base';
+  db_access.name = '~/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/config/Info.dpl';
+% $$$   db_access.user = 'myself';
+% $$$   db_access.pass = 'top_secret';
+% $$$   db_access.driver = 'org.postgresql.Driver';
+
+end
```

Comment:
Another 'git' question:

Originally, I forked your repository to my gitHub and then I created a local branch. But (at least I couldn't fin how) I was not able to do the pull request from the branch. I had to merge the branch to the trunk before doing the PR. So when you suggest to do reset master <file>, I guess it will not work as you expect right? The "master" is the version just before the PR which is different from the forked version...